tests/ti_deps/deps/test_pool_slots_available_dep.py

```diff
@@ -0,0 +1,76 @@
+# -*- coding: utf-8 -*-
+#
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+import unittest
+
+from mock import Mock, patch
+
+from airflow import settings
+from airflow.models import Pool
+from airflow.ti_deps.deps.pool_slots_available_dep import PoolSlotsAvailableDep, \
+    STATES_TO_COUNT_AS_RUNNING
+from tests.test_utils import db, decorators
+
+
+class PoolSlotsAvailableDepTest(unittest.TestCase):
+    def setUp(self):
+        db.clear_db_pools()
+        session = settings.Session()
```

Comment:
Please use `with create_session() as session:` here as well.

Reply:
roger that.